pilot/lib/pilot.rb

```diff
@@ -1,13 +1,15 @@
+require "fastlane_core"
```

Comment:
Maybe add a comment here, that it has to be imported before the other files, otherwise we'll forget in a few weeks and might move it by mistake